coredns/plugin/handler_test.go

```diff
@@ -975,10 +975,12 @@ func testClusterSetIP() {
 
 		si := newServiceImport(namespace1, service1, mcsv1a1.ClusterSetIP)
 		si.Spec.IPs = []string{clusterSetIP}
-		si.Spec.Ports = []mcsv1a1.ServicePort{port1, port2}
 
 		t.lh.Resolver.PutServiceImport(si)
 
+		si.Spec.Ports = []mcsv1a1.ServicePort{port1, port2}
+		t.lh.Resolver.PutServiceImport(si)
```

Comment:
Could you add a comment here explaining why `PutServiceImport` needs to be called twice? I’m concerned that in a few months’ time someone might see this and remove the earlier `PutServiceImport` call since it appears redundant.

Reply:
I removed the two calls from here as the scenario is handled in _clusterip_service_test.go_.